Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM docker.io/library/rust:latest as builder
```

Comment:
Could this be:

```
FROM rust:1.76.0-bookworm as builder
```

Reply:
@casey I've changed it on https://github.com/casey/intermodal/pull/533/commits/9e69c826eab7fe24f9ddbe30519413ace16874b1